dashboard/Dockerfile

```diff
@@ -16,6 +16,9 @@ WORKDIR /app
 ENV NEXT_TELEMETRY_DISABLED 1
 ENV NEXT_PUBLIC_NHOST_PLATFORM false
 ENV NEXT_PUBLIC_NHOST_BACKEND_URL http://localhost:1337
+ENV NEXT_PUBLIC_NHOST_MIGRATIONS_URL http://localhost:9693
```

Comment:
Is it possible to change the default ports through the CLI @elephant3?

I think I can't use hardcoded URLs like this, can I?

Reply:
yes, it's possible (see `--console-api-port`  flag)

```
nhost up --help
Initialize a local Nhost environment for development and testing.

Usage:
  nhost dev [-p port] [flags]

Aliases:
  dev, up

Flags:
      --console-api-port uint32   Port for serving hasura migrate API (default 9693)
      --console-port uint32       Port for hasura console (default 9695)
      --db-port uint32            Port for database (default 5432)
      --graphql-port uint32       Port for graphql server (default 8080)
      --hasuracli string          User-defined path for hasura-cli binary
  -h, --help                      help for dev
      --mailhog-port uint32       Port for mailhog UI (default 8025)
      --minio-s3-port uint32      S3 port for minio (default 9000)
      --no-browser                Don't open browser windows automatically
  -p, --port uint32               Port for dev proxy (default 1337)
      --smtp-port uint32          Port for smtp server (default 1025)
      --start-timeout duration    Timeout for starting services (default 10m0s)

Global Flags:
  -d, --debug           Show debugging level logs
      --domain string   Auth domain - for internal testing (default "nhost.run")
  -j, --json            Print JSON formatted logs
      --path string     Current working directory to execute CLI in (default "/Users/siarheilipchyk/code/be/functions/ext-loki-logger")
```